libnetwork/drivers/ipvlan/ipvlan.go

```diff
@@ -64,18 +64,16 @@ type network struct {
 
 // Register initializes and registers the libnetwork ipvlan driver.
 func Register(r driverapi.Registerer, config map[string]interface{}) error {
```

Comment:
We're currently opening `local-kv.db` from a few different places. For a follow-up PR: config should be replaced with a `datastore.DataStore` here.